next-desolates-map/tailwind.config.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
     theme: {
         extend: {
             fontFamily: {
-                cool: 'Share Tech Mono, monospace'
+                cool: 'Share Tech Mono, monospace',
+                serif: ['Nasalization Rg', 'serif'],
```

Comment:
Instead of `serif: ['Nasalization Rg', 'serif'],`

I would do `nasalization: ['Nasalization Rg', 'serif'],`